src/CashuWallet.ts

```diff
@@ -682,6 +690,31 @@ class CashuWallet {
 		const { blindedMessages, rs, secrets } = this.createBlindedMessages(amounts, keysetId, counter);
 		return { blindedMessages, secrets, rs };
 	}
+
```

Comment:
It would probably make sense to move this to the cashu-crypto lib too. We can do it in a later PR though